src/ql/arch/cc/info.cc

```diff
@@ -587,7 +720,22 @@ void Info::post_process_platform(
  */
 void Info::populate_backend_passes(pmgr::Manager &manager, const utils::Str &variant) const {
 
-    // Mimic original CC backend.
+    // Remove passes we don't need/want.
+    // FIXME: bit of a hack, and invalidates https://openql.readthedocs.io/en/latest/gen/reference_architectures.html#default-pass-list
+    const auto &prescheduler = com::options::global["prescheduler"];
+    if (prescheduler.as_bool() && prescheduler.get_default() == prescheduler.as_str()) {    // prescheduler enabled implicitly
```

Comment:
I suppose I should have looked it up when this came up over e-mail. The correct way to check for this is `!prescheduler.is_set()`; is_set() returns true iff the option was explicitly configured by the user.